### DISCUSSION TITLE
Implementing 5Sims instead of sms-activate

### DISCUSSION THREAD

**Discussion by ProfoundPenguin** [leoshabit/Auto-Gmail-Creator#33] [General]
Hey, guys

I have successfully implemented 5Sims on the script, but the issue is 99% of their numbers bounce, they are either used or blocked.
I am using this API link, please tell me if I am wrong "https://5sim.net/v1/user/buy/activation/brazil/any/google"

I have tried many countries and service providers, still no success.

cheers.

**Comment by xD-E-A-T-Hx** [comment]
I'd assume the cheaper the number the more likely people are to use it. These companies only have set amount of numbers and if everyone using them then they are likely to bounce. 

If you go for a more expensive number, the more likely for success. Then it ends up being the case of us it cheaper to buy to gmail accounts than to pay for an sms number.

**Reply by zrrGUJJAR** [reply]
Am need some help do you please share your email address.
Thanks

**Comment by ProfoundPenguin** [comment]
Services like 5sim, don't they offer numbers that aren't used or aren't used too many times for websites like Google?
I thought that was their whole deal, providing numbers that should work.

I thought there was something wrong with the way I was calling for the API, maybe I was asking for the wrong service. 

You might have more XP, is sms-activate.org more reliable?

**Reply by xD-E-A-T-Hx** [reply]
Not at all. I only have ever uses sms-activate. They have google specific numbers also. 
If I'm using American proxies I use American numbers. If I'm using Canadian proxies I use Canadian numbers. I feel i am more often getting a number that can't be used than can. 

So it could just be the luck of the draw. If I get time next week I'll look at this other number supplier see if it makes a difference for me

**Reply by ProfoundPenguin** [reply]
Ah, sure man thanks

**Reply by brockmisner** [reply]
sms pool is really good all the number work

**Reply by zrrGUJJAR** [reply]
> sms pool is really good all the number work

Am need some help do you please share your email.
Thanks

**Comment by mauricewasswa** [comment]
***@***.***
